Dockerfile

```diff
@@ -0,0 +1,43 @@
+
+# Uses alpine as base
```

Comment:
Should this be split into a build-env and execution-env so we get the smallest possible container?